user/models.py

```diff
@@ -0,0 +1,67 @@
+from django.db import models
+from order.models import *
+from product.models import *
+from board.models import *
+
+
+class Member(models.Model):
+    email             = models.EmailField(max_length=100)
+    nickname          = models.CharField(max_length=20)
+    privacy_agreement = models.BooleanField()
+    anonymous         = models.BooleanField()
+    random_number     = models.CharField(max_length=45)
+    password          = models.CharField(max_length=100)
+    board             = models.ManyToManyField('board.Board', through='board.BoardLike')
+    comment           = models.ManyToManyField('board.Comment', through='board.CommentLike')
```

Comment:
```suggestion
    comment           = models.ManyToManyField('board.Comment', through='board.CommentLike')
```
이요소가 보드에 있어야 할까요? 유저에 있어야 할까요?

Reply:
그러면 보드로 가서 연결을 해야하나요? 제가 알기로는 many to many를 사용할때는 어느 테이블에 넣어도 연결이된다고 배웠는데 아닌가요? 특정 테이블에 넣어야하나요?